.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: java
 jdk:
   - oraclejdk8
-script: mvn integration-test -Dcheckstyle.failOnViolation=false
+script: mvn verify -Dcheckstyle.failOnViolation=false
```

Comment:
Please remove the "-Dcheckstyle.failOnViolation=false" argument, this project is in SNAPSHOT state, the failure should be fixed before release.